metaworld/envs/mujoco/multitask_env.py

```diff
@@ -101,7 +101,7 @@ def _check_env_list(self):
 
     @property
     def observation_space(self):
-        """ TODO not complete, needs support for different observation types"""
+        """TODO not complete, needs support for different observation types."""
```

Comment:
Please don't merge with TODOs. If you see a flaw which could be addressed later, file an issue tracking it.